database/index_room.js

```diff
@@ -30,15 +37,16 @@ var getAllRooms = function(callback) {
       console.log(`Error getting rooms info from DB`)
```

Comment:
Very clean code! do you think you could send err in the callback function as well?